iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/Networks.java

```diff
@@ -450,4 +450,25 @@ public static AbstractReportNodeContext allowReportNodeContextMultiThreadAccess(
         }
         return newReportNodeContext != null ? newReportNodeContext : reportNodeContext;
     }
+
+    /**
+     * This method replaces the "input" status of switches, tap changer position and shunt section count by their solved values if they are present.
+     */
+    public static void applySolvedValues(Network network) {
+
+        // Two-windings transformers
+        network.getTwoWindingsTransformerStream().map(RatioTapChangerHolder::getRatioTapChanger).filter(Objects::nonNull).forEach(TapChanger::applySolvedValues);
```

Comment:
Can you add a `applySolvedValues()` method on two and three winding transformers? It would be easier to use.